src/main/java/com/movie/store/security/AddAdminService.java

```diff
@@ -0,0 +1,36 @@
+package com.movie.store.security;
+
+import com.movie.store.model.Role;
+import com.movie.store.model.User;
+import com.movie.store.service.interfaces.RoleService;
+import com.movie.store.service.interfaces.UserService;
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AddAdminService {
+    private static final String USER = "USER";
```

Comment:
I think USER_ROLE_NAME will be better here)

Reply:
> I think USER_ROLE_NAME will be better here)

\+